### PR TITLE
[Operators] v2 operators addition

### PR DESCRIPTION
**Pull request by StrongSpoon**
1. reorganize the source code. single operators and fused operators are placed in two directories, in order to add operators from silicon flow.
2. fix the bug of backward error. group_norm, tanh and sigmoid can pass all test cases new.
3. implement binary pointwise ops using codegen, including div, mul, pow, bitwise_and, bitwise_or.

### CODE REVIEW

File: tests/flag_gems/op_accu_test.py [hunk 1]
```diff
@@ -143,6 +143,26 @@ def test_accuracy_addmm(M, N, K, alpha, beta, dtype):
...
+    )
+    inp2 = torch.randint(
+        low=-0x7FFF, high=0x7FFF, size=shape, dtype=dtype, device="cuda"
+    )
+
+    ref_out = torch.bitwise_and(inp1, inp2)
+    with flag_gems.use_gems():
+        res_out = torch.bitwise_and(inp1, inp2)
+
+    torch.testing.assert_close(res_out, ref_out, atol=0, rtol=0)
```
Comment (tongxin) [inline tests/flag_gems/op_accu_test.py]: why not testing eq?

File: src/flag_gems/ops/tanh.py [hunk 2]
```diff
@@ -21,9 +21,9 @@ class Tanh(torch.autograd.Function):
     @staticmethod
     def forward(ctx, A):
         logging.debug("GEMS TANH FORWARD")
-        O = tanh_forward(A)
+        O = tanh_forward(A.to(torch.float32))
         ctx.save_for_backward(O)
```
Comment (tongxin) [inline src/flag_gems/ops/tanh.py:25]: do we always need to do save for backward regardless gradient context?

File: tests/flag_gems/op_accu_test.py (hunk 1 above)
Comment (StrongSpoon) [reply tests/flag_gems/op_accu_test.py]: because I forgot orz

File: src/flag_gems/ops/tanh.py (hunk 2 above)
Comment (StrongSpoon) [reply src/flag_gems/ops/tanh.py:25]: no, we only save variants needed in backward

File: tests/flag_gems/op_accu_test.py (hunk 1 above)
Comment (StrongSpoon) [reply tests/flag_gems/op_accu_test.py]: done

**Pull request merged**